src/connectedk8s/setup.py

```diff
@@ -36,7 +36,8 @@
 # TODO: Add any additional SDK dependencies here
 DEPENDENCIES = [
     'kubernetes==11.0.0',
-    'pycryptodome==3.9.8'
+    'pycryptodome==3.9.8',
+    'cryptography>=2.3.1,<3.0.0'
```

Comment:
This conflicts with `azure-cli-core`'s dependency on `cryptography` (https://github.com/Azure/azure-cli/pull/15687).

`cryptography>=2.3.1,<3.0.0` has known security issues. It should use the one provided by `azure-cli-core`.

Reply:
hi @akashkeshari for dependencies already in azure-cli-core, extension no need to explicitly call out the dependency again like cryptography. It will cause problem as we discussed in email thread. pls help to remove it.